qiskit/providers/ibmq/experiment/analysis_result.py

```diff
@@ -58,6 +58,7 @@ def __init__(
             tags: Optional[List[str]] = None,
             result_uuid: Optional[str] = None,
             backend_name: Optional[str] = None,
+            verified: Optional[bool] = None
```

Comment:
I don't know if we want to default this to False here or not, the server side will default it based on the `quality` value passed in, so might as well just leave it default to None in the client so it's not included in the request body.